nextflow_schema.json

```diff
@@ -580,6 +580,13 @@
                     "description": "Institutional config URL link.",
                     "hidden": true,
                     "fa_icon": "fas fa-users-cog"
+                },
+                "test_data_base": {
+                    "type": "string",
+                    "default": "https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq3",
```

Comment:
Why `rnaseq3` ?

Reply:
Created a subset of `rnaseq` which contains only the data actually used in the tests